clang/lib/Analysis/FlowSensitive/Models/UncheckedOptionalAccessModel.cpp

```diff
@@ -129,19 +215,19 @@ auto inPlaceClass() {
 
 auto isOptionalNulloptConstructor() {
   return cxxConstructExpr(
-      hasOptionalType(),
+      hasOptionalOrDerivedType(),
```

Comment:
Here and below -- now that this matcher is more expensive, please move until after the less expensive matchers.

Reply:
Done.